doc/guides/maintaining-V8.md

```diff
@@ -87,17 +87,31 @@ includes the following branches<sup>1</sup>:
    </td>
   </tr>
   <tr>
-   <td>Node.js v7.x
+   <td>Node.js v8.x
```

Comment:
Nit: I think we should avoid `v` altogether when talking about version numbers for Node.js, so I'd prefer this as `Node.js 8.x` rather than `Node.js v8.x`.

Reply:
Done.